lib/Command/Configure.php

```diff
@@ -110,12 +112,101 @@ private function configureSignal(InputInterface $input, OutputInterface $output)
 	private function configureSms(InputInterface $input, OutputInterface $output) {
 		$helper = $this->getHelper('question');
 
-		$providerQuestion = new Question('Please choose a SMS provider (websms, playsms, clockworksms, puzzelsms, ecallsms, voipms, voipbuster, huawei_e3531, spryng, sms77io, ovh, clickatellcentral, clicksend): ', 'websms');
+		$providerQuestion = new Question('Please choose a SMS provider (customsms, websms, playsms, clockworksms, puzzelsms, ecallsms, voipms, voipbuster, huawei_e3531, spryng, sms77io, ovh, clickatellcentral, clicksend): ', 'customsms');
 		$provider = $helper->ask($input, $output, $providerQuestion);
 
 		/** @var SMSConfig $config */
 		$config = $this->smsGateway->getConfig();
 		switch ($provider) {
+			case 'customsms':
+				$config->setProvider($provider);
+				/** @var CustomSMSConfig $providerConfig */
+				$providerConfig = $config->getProvider()->getConfig();
+
+				ReTypeUrl:
+				$urlQuestion = new Question('Please enter the web service URL: ');
+				$url = $helper->ask($input, $output, $urlQuestion);
+
+				if(!filter_var($url, FILTER_VALIDATE_URL))
+				{
+					$output->writeln('Invalid URL '.$url);
+					goto ReTypeUrl;
```

Comment:
`goto` isn't good, maybe will be good use methods to do this using recursion.
Using specific method also will make more easy implement unit tests to this command;